<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndpointNodeUiWidget</class>
 <widget class="QWidget" name="EndpointNodeUiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EndpointNode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Request :</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Connection</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Content-Type :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Timeout (msec) : </string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Method :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Url :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_method">
              <item>
               <property name="text">
                <string>GET</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>POST</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DEL</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_url">
              <property name="placeholderText">
               <string>http://endpoint.url</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_timeout">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>3000</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_contentType">
              <item>
               <property name="text">
                <string>application/json</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>text/plain</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>text/html</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Authentication</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>None</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>No authentication required</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Basic</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>User :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="placeholderText">
                   <string>admin</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Pwd : </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_2">
                  <property name="echoMode">
                   <enum>QLineEdit::PasswordEchoOnEdit</enum>
                  </property>
                  <property name="placeholderText">
                   <string>admin</string>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="checkBox_preemtiveAuth">
                  <property name="text">
                   <string>Preemptive authentication</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>41</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>Bearer</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Token :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>90</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_7">
             <attribute name="title">
              <string>OAuth</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Under development</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Http codes</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Headers</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="rowCount">
             <number>100</number>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Response :</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit_response">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Console ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
